Data/FlixnChill - user data - Sheet1.csv

```diff
@@ -0,0 +1,55 @@
+Name,Age,Sex,Tagline,Photo,,,,

```

Comment:
yep . The data is fetched from Parse right now. Just added this here for backup reasons.   
https://www.parse.com/apps/flixnchill--2/collections#class/PotentialMatches